styles/nav.less

```diff
@@ -80,6 +80,8 @@ nav {
 
 // Table of content
 ul.nav.nav-stacked {
+    height: 75vh;
```

Comment:
I think it is better to change the height to 90vh so that the nav takes the full height of the page